### ISSUE TITLE
index function for value list

### ISSUE THREAD

**Issue by KaptinLin** [less/less.js#2078]
http://www.smashingmagazine.com/2014/06/12/sassy-z-index-management-for-complex-layouts/

any plan this function support?

**Comment by Hellowor1d** [comment]
This is really a good feature for z-index management ~ 
Giving my +1 to the issue 👍

**Comment by matthew-dean** [comment]
Just to be clear, this is just a function that increments each time that it's called? 

This feels like such a niche function that this is more appropriate as a `@plugin`.

**Comment by seven-phases-max** [comment]
@matthew-dean
>Just to be clear, this is just a function that increments each time that it's called?

No, it's about a function returning the index of the specified value in a list. E.g. basically like `Array.indexOf`.of JS.
